Shared/configurations/Configuration.h

```diff
@@ -30,7 +30,7 @@ class Configuration
                 bool selected,
                 bool loaded,
                 int percentDownloaded);
-  Configuration(Configuration&) = default;
+  Configuration() = default;
```

Comment:
Was this intended? I wouldn't think a Configuration with all default values is useful.

```suggestion
  Configuration() = delete;
```

Reply:
I did some more testing and realized there's no need for any of these special method definitions. I only added these originally because they were used in a container. I was certain I was getting a warning or error related to my call to `emplace_back` in `ConfigurationListModel`... No longer able to reproduce it though. Maybe simpler to just remove them and let the compiler sort things out.